site/src/get-started/local-development.md

```diff
@@ -9,12 +9,13 @@ eleventyNavigation:
 
 In this guide, we'll look at how to use `@angular/fire` to connect an Angular application with the Firebase Emulator Suite to start prototyping your apps.
 
-There are four supported emulators, all of them available at the Firebase suite workflow:
+There are five supported emulators, all of them available at the Firebase suite workflow:
 
 - [Authentication Emulator](https://firebase.google.com/docs/emulator-suite/connect_auth)
 - [Realtime Database Emulator](https://firebase.google.com/docs/emulator-suite/connect_rtdb)
 - [Cloud Firestore Emulator](https://firebase.google.com/docs/emulator-suite/connect_firestore)
 - [Cloud Functions Emulator](https://firebase.google.com/docs/emulator-suite/connect_functions)
+- [Cloud Storage Emulator](https://firebase.google.com/docs/emulator-suite/connect_storage)
```

Comment:
does not exist yet, but i expect the docs guys will use this url?

Reply:
This URL exists now.